src/main/java/frc/robot/RobotContainer.java

```diff
@@ -40,4 +46,8 @@ public RobotContainer() {
   private void configureBindings() {
     driverController.a().whileTrue(new InstantCommand(intake::deploy, intake));
   }
+
+  public static Command getAutonomousCommand(){
+    return new AutoChargeStationSequence(true, new DriveSubsystem());
```

Comment:
Can you instantiate a driveSubsystem as a member variable in this class, and then just pass that instance into this function?

Reply:
Okay, when I did this I also changed the method so it's not longer static.